.github/workflows/babel-plugin-static-check.yml

```diff
@@ -54,9 +55,6 @@ jobs:
       - name: Check types
         working-directory: ${{ env.PLUGIN_DIR }}
         run: yarn type:check
-      - name: Lint and format
```

Comment:
we don't need it anymore because `yarn run lint:root` will scan reanimated plugin files